build/test-aot.ps1

```diff
@@ -16,7 +16,7 @@ foreach ($line in $($publishOutput -split "`r`n"))
 }
 
 Write-Host "Actual warning count is: ", $actualWarningCount
-$expectedWarningCount = 65
+$expectedWarningCount = 50
```

Comment:
The outstanding warnings need breaking changes in Abstractions and MSAL.NET:
See https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/pull/178